packages/salesforcedx-vscode-core/test/vscode-integration/commands/functions/forceFunctionStart/ForceFunctionContainerlessStartExecutor.test.ts

```diff
@@ -0,0 +1,285 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { vscodeStub } from '@salesforce/salesforcedx-utils-vscode/out/test/unit/commands/mocks';
+import { expect } from 'chai';
+import * as proxyquire from 'proxyquire';
+import { assert, createSandbox, SinonSandbox, SinonStub, stub } from 'sinon';
+
+const proxyquireStrict = proxyquire.noCallThru();
+
+describe('ForceFunctionContainerlessStartExecutor unit tests', () => {
```

Comment:
similar to the container executor test, but only covering the containersless method implementations.